### PR TITLE
Added common functions to own module

### LINKED ISSUE #46: Code Duplication
There is a HUGE amount of code duplicity between the different run scripts.
This needs to be fixed if possible, such that there is less code which needs to be maintained. 

For example, see #45 and #40  , a pause in the copy was implemented for both `hejrun.py` and `sherparun.py`, and only later ported to `nnlojet.py`

### PR DESCRIPTION
**Pull request by marianheil**
This adds a new module for common functions, which is also copied to grid nodes.

Closes #46

### CODE REVIEW

**Comment by marianheil** [comment]
I have done a small test with HEJ, Sherpa & the example. However I can't test NNNNLOJets

**Comment by jcwhitehead** [comment]
This doesn't work out-of-the-box for me - I'll put a bit of time into fixing it tonight and see how it goes.

File: src/pyHepGrid/extras/grid_functions.py [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+def set_default_environment(args, lhapdf_dir):
+    """
+    Initialise default environment, such that gfal works
+    """
+    do_shell(
+        "export PYTHONPATH=${PYTHONPATH}:${DIRAC}/Linux_x86_64_glibc-2.12/lib"
+        "/python2.6/site-packages")
+    os.environ["LFC_HOST"] = "lfc01.dur.scotgrid.ac.uk"
+    os.environ["LCG_CATALOG_TYPE"] = "lfc"
+    os.environ["LCG_GFAL_INFOSYS"] = "lcgbdii.gridpp.rl.ac.uk:2170"
```
Comment (marianheil) [inline src/pyHepGrid/extras/grid_functions.py:176]: While we're on it: Can we remove these variables? I don't think they are relevant with gfal any longer.

File: src/pyHepGrid/extras/grid_functions.py [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+    os.environ['OMP_STACKSIZE'] = "999999"
```
Comment (marianheil) [inline src/pyHepGrid/extras/grid_functions.py:177]: This one is NNNNNLOJet specific? We should move it to `nnnnlorun.py`

Comment (jcwhitehead) [reply src/pyHepGrid/extras/grid_functions.py]: Done

File: src/pyHepGrid/extras/grid_functions.py (hunk 1 above)
Comment (jcwhitehead) [reply src/pyHepGrid/extras/grid_functions.py:176]: Go for it - as far as I'm aware they're not used.

**Pull request merged**